README.md

```diff
@@ -41,6 +35,11 @@ You can also pause the replay by pressing `z`. When the replay is paused this wa
 Mod can be buit with `cargo` using the `nightly-i686-pc-windows-msvc` toolchain.  
 When building from source please remember to add the `--release`/`-r` flag.
 
+```
+rustup toolchain install nightly-i686-pc-windows-msvc
```

Comment:
It's for newer contributors like myself. If I tried building from a new installation of rust, it would complain that the nightly i686 toolchain is not installed. Other new contributors might have the same issue. If installing it took more steps I would include a link to those steps from an official source instead.

Of course, I could make a link to rustup toolchains in the paragraph above which would remove this command and retain clarity.